app/mailers/notifier.rb

```diff
@@ -0,0 +1,10 @@
+class Notifier < ActionMailer::Base
+  def statistic statistic
+    @daily_statistic = statistic
+
+    mail subject: t('digest.email_header.subject', date: @daily_statistic.for_today),
+         to:      User.all.map {|user| user.email},
```

Comment:
@vlasiak, I suggest to put a responsibility to know about the recipients to `daily_statistic` object and get rid of the usage of `User` model here (and the logic of obtaining of emails).